Content.Server/EntityEffects/EffectConditions/TotalHunger.cs

```diff
@@ -15,9 +15,10 @@ public sealed partial class Hunger : EntityEffectCondition
 
     public override bool Condition(EntityEffectBaseArgs args)
     {
-        if (args.EntityManager.TryGetComponent(args.TargetEntity, out HungerComponent? hunger))
+        if (args.EntityManager.TryGetComponent(args.TargetEntity, out HungerComponent? hunger) &&
+            args.EntityManager.SystemOrNull<HungerSystem>() is { } hungerSystem)
```

Comment:
I don't know if `args.EntityManager.SystemOrNull<HungerSystem>()` is the right thing to do here, but I am not sure how else to access the system.

Reply:
The only `GetEntitySystem` I could find is on `IEntitySystemManager` which, I guess, would need to be injected, but I assume `[Dependency]`s can't be injected into non-systems like this.

Instead, I've changed it to just do `args.EntityManager.System<...>()`, based on
https://github.com/space-wizards/space-station-14/blob/fdd713a0f0e5278ef9ae24c93dcbc7c8e8094e15/Content.Server/EntityEffects/EffectConditions/HasTagCondition.cs#L21
doing that same thing.

If this is still not the right thing, do you mind providing me with a bit more guidance?